### ISSUE TITLE
Can't get two array data by one set 

### ISSUE THREAD

**Issue by ninozhang** [rc0x03/node-osmosis#15]
Code:
```
    .set({
        'otherCityLink[]': 'li a@href'
        'otherCity[]': 'li a > span'
    })
```
the second field of the result will be null.
and have to change the code to:
```
    .set({
        'otherCityLink[]': 'li a@href'
    })
    .set({
        'otherCity[]': 'li a > span'
    })
```

**Comment by rc0x03** [comment]
This issue is due to a bug in the new `.set` nesting capability. The issue should be fixed now in version 0.0.6.

**Comment by robert26bussard51** [comment]
RÖB
